.github/workflows/wheels.yml

```diff
@@ -0,0 +1,76 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Stable release build
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+    branches:
+      - master
```

Comment:
does this mean we also run for all pull requests merged into master?

Reply:
We have to figure out indeed when to run this (e.g. cron, on master etc) but that's something we could discuss at the final stages once we know this could work in general :)

Reply:
I have included the run for all pull requests merged into `master` branch for debugging until this is working.

However, I think that it could be interesting to avoid issues like #17680 or #17767.